src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -57,9 +64,15 @@ public HelpWindow() {
      */
     public void show() {
         logger.fine("Showing help page about the application.");
-        Image image = new Image("file:Commands.png", 637, 886, true, false);
         imageView.setImage(image);
 
+        FlowPane root = new FlowPane(Orientation.HORIZONTAL);
+        root.getChildren().addAll(imageView);
+        ScrollPane sp = new ScrollPane();
+        sp.setContent(root);
+        Scene scene = new Scene(sp, 550, 400);
+        stage.setScene(scene);
```

Comment:
This can be added to the helpwindow fxml instead. There is no need to set up new scene and pane